win32ss/user/ntuser/windc.c

```diff
@@ -41,7 +41,8 @@ DceCreateDisplayDC(VOID)
 {
   UNICODE_STRING DriverName = RTL_CONSTANT_STRING(L"DISPLAY");
 
-  co_IntGraphicsCheck(TRUE);
+  if (!co_IntGraphicsCheck(TRUE))
+    KeBugCheckEx(VIDEO_DRIVER_INIT_FAILURE, 0, 0, 0, USER_VERSION);
```

Comment:
Direct bugcheck? What would happen if instead you just failed the `DceCreateDisplayDC` call by returning NULL?

Reply:
I already tried that. If I return NULL, error is propagated through all callers.
From user point of view, screen stays black and nothing happens.

Reply:
thanks :)